src/App.css

```diff
@@ -1,38 +1,22 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
-  }
+body {
```

Comment:
- [ ] You should remove the outline on the page and implement only the page not the white space around it, also consider using the same fonts as in the original design, so your app is designed correctly

|Your design|Zeplin design|
|-|-|
|![Screenshot_20220625_180813](https://user-images.githubusercontent.com/25227022/175783986-0aa84160-b21c-4868-b6f9-65da56c4e692.png)|![Screenshot_20220625_181030](https://user-images.githubusercontent.com/25227022/175783993-7a807aae-884a-49a7-a1b5-e4e76f688958.png)|